### DISCUSSION TITLE
how the it will behave if a app is registered in both SHARED_APPS and TENANT_APPS?

### DISCUSSION THREAD

**Discussion by praburocking** [django-tenants/django-tenants#620] [Q&A] [answered]
like to know the behaviour of the framework when an app is registered in both SHARED_APPS and TENENT_APPS list in settings.py.
will the model will be created in both shared schema and tenant schema?
how the persisting of the model works?

**Comment by itzomen** [comment, marked as answer]
Yup, it will be created in the public and tenants schema...
But it is sufficient to place an app you want on both schemas in the shared apps
